include/zephyr/bluetooth/gatt.h

```diff
@@ -1940,9 +1945,27 @@ struct bt_gatt_read_params {
 			bool variable;
 		} multiple;
 		struct {
-			/** Attribute handle to start reading from. */
+			/** @brief requested start attribute handle number.
+			 *
+			 * @details The starting handle is set to the starting point of the range
+			 * over which this read should be performed. For example, this could be
+			 * set to @ref BT_ATT_FIRST_ATTRIBUTE_HANDLE to set the starting point of
+			 * the range at the beginning of the GATT database, or to the starting
+			 * handle of a service after discovery.
+			 *
+			 * This value is also automatically incremented by the stack after
+			 * processing each matching handle-value pair returned by the server.
```

Comment:
```suggestion
			 * @note This value is automatically incremented by the stack after
			 * processing each matching handle-value pair returned by the server.
```

I think having a note might be useful here (and remove the "also" as it's just noise).

Reply:
I would avoiding adding this as a note. It is just a detail in my mind. It looks fine in the HTML output.